### PR TITLE
Update Braker3 

### PR DESCRIPTION
**Pull request by rlibouba**
Hello, 
The Braker3 tool has been updated to version 3.0.7.
I opened this PR to update the toolshed. 
Have a nice day!

### CODE REVIEW

**Comment by bgruening** [comment]
Thanks for taking care of this. Should we maybe also support older version? I guess when we merge this, older versions will not work anymore?

**Comment by sanjaysrikakulam** [comment]
The change in this PR will prevent this tool's old versions from getting scheduled.

If you want to include the new version to get scheduled using a new container or parameters and conditions in addition to the old version, you can add the changes as a new entry to the tools list instead.

**Comment by sanjaysrikakulam** [comment]
Also, please update your branch; it seems outdated.

**Comment by rlibouba** [comment]
I feel stupid, I closed the PR by updating my branch and wanted to resolve conflicts. I'm really sorry. Is it possible to reopen it? 

Thanks @sanjaysrikakulam  for your feedback.

**Comment by rlibouba** [comment]
I feel stupid. I closed the PR in an attempt to update my branch and resolve conflicts. Sorry about that.

Thanks @sanjaysrikakulam for the feedback. 
Thanks Eva for your help.

**Review by bgruening — APPROVED** [review]

File: files/galaxy/tpv/tools.yml [hunk 1]
```diff
@@ -1060,21 +1060,21 @@ tools:
...
-
-  toolshed.g2.bx.psu.edu/repos/genouest/braker3/braker3/3.0.3.*:
     mem: 4
-    cores: 6
     params:
-      singularity_container_id_override: docker://quay.io/bgruening/braker3:v.1.0.4
-    scheduling:
-      require:
-        - singularity
+      singularity_enabled: true
```
Comment (abretaud) [inline files/galaxy/tpv/tools.yml:1067]: I think the +galaxy suffixes are needed for the tool versions here, right?
Looks like braker3 is not working at EMBO training

Comment (abretaud) [reply files/galaxy/tpv/tools.yml:1067]: https://github.com/usegalaxy-eu/infrastructure-playbook/pull/1094

**Pull request merged**